models/issue_comment.go

```diff
@@ -566,6 +600,45 @@ func CreateRefComment(doer *User, repo *Repository, issue *Issue, content, commi
 	return err
 }
 
+// ClearPullPushComent clear all the push commit on issue since fore push
+func ClearPullPushComent(issue *Issue) error {
+	_, err := x.Delete(&Comment{
+		Type:    CommentTypePullPushCommit,
+		IssueID: issue.ID,
+	})
+	return err
+}
+
+// CreatePullPushComment creates a commit when push commit to a pull request.
+func CreatePullPushComment(doer *User, repo *Repository, issue *Issue, content, commitSHA, repoFullName string) error {
```

Comment:
If `repo` is already an argument, why does `repoFullName` need to be another argument?

Also, would it make more sense to have a `*Commit` argument instead of `content` and `commitSHA`?

Reply:
done.